frontend/packages/console-app/src/components/quick-starts/utils/quick-start-typings.ts

```diff
@@ -0,0 +1,21 @@
+export type QuickStartItem = {
+  iconURL: string;
+  altIcon?: string;
+  id: string;
+  name: string;
+  duration: number;
+  description: string;
+  prerequisites?: string[];
+};
+
+export type QuickStartStatus = {
```

Comment:
`QuickStartStatus` is named the same as the enum in `quick-start-status.ts`.

Can we not export this? I don't see where this is being used outside of this file.